packages/shared-ux/page/analytics_no_data/types/index.d.ts

```diff
@@ -9,12 +9,14 @@ import {
   KibanaNoDataPageServices,
   KibanaNoDataPageKibanaDependencies,
 } from '@kbn/shared-ux-page-kibana-no-data-types';
+import { Observable } from 'rxjs';
 
 /**
  * A list of services that are consumed by this component.
  */
 export interface Services {
   kibanaGuideDocLink: string;
+  customBranding: { showPlainSpinner?: boolean; hasCustomBranding$: Observable<boolean> };
```

Comment:
Remove `showPlainSpinner` from service definition

Reply:
Ok wow - so showPlainSpinner is set within the `<AnalyticsNoDataPageComponent />` so it doesn't actually need to be specified as a service because it's dependent only on hasCustomBranding$ 🤯